lib/templates.js

```diff
@@ -118,18 +118,13 @@ function _getTitleFromServiceName(srv) {
 const fCreateGroupsTemplateForService = (srv, srvDefinition) => {
     const ordExtensions = fReadORDExtensions(srvDefinition);
 
-  let fullyQualifiedServiceName = srv;
-  if (!srv.includes(global.capNamespace)) {
-    fullyQualifiedServiceName = global.capNamespace + "." + srv;
-  }
```

Comment:
No need for generating fullyQualifiedServiceName: just take the name of the service as it is generated by the cds compiler.

cc: @Fannon @zongqichen